README.md

```diff
@@ -40,7 +40,7 @@ Run truffle test:
 ```shell script
 truffle compile
 truffle migrate
-truffle test
+npm run truffle:test
```

Comment:
Any issue about previous command?

Reply:
There's a folder named `test-maintenance` in ./test which is for hardhat test. `truffle test` will include that and report an error. So I add `truffle:test: truffle test ./test/*.js` in package.json so `npm run truffle:test` will work.